chromium_src/chrome/common/chrome_features.cc

```diff
@@ -18,12 +18,16 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
     {kDesktopPWAsLinkCapturing, base::FEATURE_DISABLED_BY_DEFAULT},
 #endif
+    {kPrivacyGuide3, base::FEATURE_DISABLED_BY_DEFAULT},
 #if BUILDFLAG(IS_ANDROID)
```

Comment:
is there a reason why we're not grouping these together? It's a bit odd to have multiple `IS_ANDROID` blocks and would be easier to read I think if the `!IS_ANDROID` was an `else` for it

Reply:
Personally, I find it easier to look for things in these overrides when they are alphabetical and the guards match the upstream declarations.